<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>instructor</class>
 <widget class="QWidget" name="instructor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1169</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1131</width>
     <height>571</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>20</y>
       <width>171</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>logout</string>
     </property>
    </widget>
    <widget class="QLabel" name="no">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Instructor id :</string>
     </property>
    </widget>
    <widget class="QLabel" name="no_2">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>181</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string> INFORMATION</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>211</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Courses</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>211</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Registred courses</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>901</width>
       <height>501</height>
      </rect>
     </property>
     <property name="title">
      <string>Instructor information </string>
     </property>
     <widget class="QLabel" name="label_no">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>290</y>
        <width>120</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Salary : </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>120</width>
        <height>86</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>NAME :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>230</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Department :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_ad">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>140</y>
        <width>281</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="salary">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>290</y>
        <width>501</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_bolum">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>521</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="no2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>160</y>
        <width>141</width>
        <height>86</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Instructor no :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>110</y>
        <width>121</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>back</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>861</width>
       <height>481</height>
      </rect>
     </property>
     <property name="title">
      <string>Courses</string>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>831</width>
        <height>411</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>COURSE ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SECTION ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>SEMESTER</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>YEAR</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>20</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>40</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>back</string>
     </property>
    </widget>
    <widget class="QComboBox" name="scourse">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>881</width>
       <height>431</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>STUDENT ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>STUDENT NAME</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>STUDENT DEPART</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>STUDENT CGPA</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>STUDENT GRADE</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
